CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera.swift

```diff
@@ -319,12 +319,14 @@ fileprivate extension Camera {
             completion(.success(videoDevice))
         case .notDetermined:
             AVCaptureDevice.requestAccess(for: .video) { granted in
+                AnalyticsManager.track(event: .cameraPermissionShown, screenName: .cameraPermissionView)
```

Comment:
[Clarification]: @ValentinaIancu-Gini maybe the 1st tracking event should be as well in main thread? 🤔

Reply:
I think it doesn't matter actually. I fully don't understand that switch to the main thread.
I personally think we should let it as it is since it should be done on the background thread and mixpanel is actually also handling the events on the background thread.
I will move the other event to be outside of `DispatchQueue.main.async {`.

Reply:
Events are not related to UI updates, so its ok to keep them not in main
The only possible problem - events order. If some events will be sent concurrently, but not sure its a case here.
Good practice to keep sequenced events flow will be to send all of them from same queue

Reply:
@mrkulik I checked the order of the events and is important to be in a certain order, that's why I've added both of them on the main thread.